test/utils.test.ts

```diff
@@ -1,8 +1,8 @@
 import test from "ava";
-import { toLanguageObject, getDefaultLanguageString } from "../lib/entities/utils";
+import { toLanguageObject, getDefaultLanguageString } from "../lib/utils";
 
 const toLanguageObjectTest = test.macro((t, input, expected) => {
-  const result = toLanguageObject(input);
+  const result = toLanguageObject(input as string);
```

Comment:
`as string` looks like a hack: the unit test on line 15 passes a `Map` as input and not a string.